google/cloud/forseti/services/inventory/storage.py

```diff
@@ -930,6 +931,33 @@ def get_latest_inventory_index_id(cls, session):
             inventory_index.id)
         return inventory_index.id
 
+    @classmethod
+    def get_inv_index_id_by_scanner(cls, session, scanner_index_id):
+        """List all inventory index entries.
+
+        Args:
+            session (object): Database session
+            scanner_index_id (int): id of the scanner in scanner_index table
+
+        Returns:
+            int64: inventory index id
+        """
+
+        inventory_index = (
+            session.query(InventoryIndex)
+            .outerjoin(ScannerIndex,
+                       ScannerIndex.inventory_index_id == InventoryIndex.id)
+            .filter(
+                and_(ScannerIndex.id == scanner_index_id,
+                     or_(InventoryIndex.inventory_status == 'SUCCESS',
+                         InventoryIndex.inventory_status == 'PARTIAL_SUCCESS'))
+            ).order_by(InventoryIndex.id.desc()).first())
+        session.expunge(inventory_index)
+        LOGGER.info(
+            'Latest success/partial_success inventory index id is: %s',
```

Comment:
This message would be more accurate as `Found inventory_index_id %s from scanner_index_id %s.'